apps/fluent-tester/src/FluentTester/TestComponents/Badge/BadgeTest.tsx

```diff
@@ -65,6 +65,14 @@ export const BasicBadge: React.FunctionComponent = () => {
       <Text>Compressible badge</Text>
       <CompressibleBadge />
       <CompressibleBadge text="Basic compressible" />
+      <Text>Presence Badge</Text>
+      <PresenceBadge presence="available" />
```

Comment:
Do all the presence badges to svgs? we should gate them with the svgIconsEnabled check if so

Reply:
yes, you're right. Thanks